src/Integration.UnitTests/Telemetry/TelemetryManagerTests.cs

```diff
@@ -452,5 +452,160 @@ private static void CheckExpectedLanguages(TelemetryData data, params string[] e
         }
 
         #endregion
+
+        [TestMethod]
+        public void Sequence_InstallFollowedByMultipleUpdates()
```

Comment:
I'm not particularly keen on this test, but it wasn't obvious what the expected behaviour on the day of installation/day 2 should be so I wrote this test to describe the expected behaviour.

The key takeaway is that `days_of_use` can be at most one greater than `days_since_installation`, since on the day of installation `days_since_installation` is zero and `day_of_use` is one.